.github/workflows/tests.yml

```diff
@@ -16,14 +16,30 @@ jobs:
 
     steps:
     - uses: actions/checkout@v4
-    - name: Set up Python ${{ matrix.python-version }}
+
+    # Python tests
+    - name: '[py] Set up Python ${{ matrix.python-version }}'
       uses: actions/setup-python@v5
       with:
         python-version: ${{ matrix.python-version }}
         cache: "pip"
-    - name: Install dependencies
+    - name: '[py] Install dependencies'
       run: |
         python -m pip install --upgrade pip
-        python -m pip install tox tox-gh-actions
-    - name: Run tests
+        python -m pip install tox tox-gh-actions django_cypress
+    - name: '[py] Run unit tests'
       run: tox
+
+    # JS tests
```

Comment:
Extra steps added to the CI workflow so we test also the JS code. For clarity I prefixed individual steps with either `[py]` or `[js]`.